xcode/Subconscious/Shared/Services/Noosphere/NoosphereService.swift

```diff
@@ -407,8 +407,14 @@ actor NoosphereService:
         try await self.sphere().traverse(petname: petname)
     }
     
-    func sphere(did: Did) async throws -> Sphere {
-        try Sphere(noosphere: self.noosphere(), identity: did.did)
+    /// Intelligently open a sphere by traversing or, if this is our address, returning the default sphere.
+    func sphere(address: Slashlink) async throws -> Sphere {
```

Comment:
Should we switch on `peer` and also handle the did case?

Reply:
Handled.